WooCommerce/Classes/ViewRelated/Orders/Order Details/Payment Section/Refunds/RefundedProductsViewController.swift

```diff
@@ -177,6 +186,44 @@ private extension RefundedProductsViewController {
 }
 
 
+// MARK: - Action Handlers
+//
+extension RefundedProductsViewController {
+
+    @objc func pullToRefresh() {
+        ServiceLocator.analytics.track(.refundedProductsPulledToRefresh)
+        let group = DispatchGroup()
```

Comment:
nit: are we planning on syncing other data? if not, might be easier to read without the `DispatchGroup`:

```
syncOrder { [weak self] _ in
  self?.refreshControl.endRefreshing()
}
```